hello-world.md

```diff
@@ -131,7 +131,7 @@ onto which video is rendered, you can set the player's target `SurfaceView`,
 `setVideoSurfaceView`, `setVideoTextureView`, `setVideoSurfaceHolder` and
 `setVideoSurface` methods respectively. You can also use `PlayerControlView` as
 a standalone component, or implement your own playback controls that interact
-directly with the player. `SimpleExoPlayer`'s `setTextOutput` and `setId3Output`
+directly with the player. `SimpleExoPlayer`'s `addTextOutput` and `setId3Output`
```

Comment:
Please also fix `setId3Output` to be `addMetadataOutput`. Thanks!

Reply:
updated in issues #6387 . Apologies i am not very much experienced with github, so i used github gui which made 2 PRs